test/env.ts

```diff
@@ -2,3 +2,4 @@ process.env.LOG_LEVEL = 'fatal';
 process.env.RIVIERE = 'false';
```

Comment:
This line should be removed with the new enabled property

Reply:
Done